x-pack/plugins/cases/public/client/api/index.ts

```diff
@@ -32,6 +32,7 @@ export const createClientAPI = ({ http }: { http: HttpStart }): CasesUiStart['ap
         getCasesStatus({ http, query, signal }),
       getCasesMetrics: (query: CasesMetricsRequest, signal?: AbortSignal): Promise<CasesMetrics> =>
         getCasesMetrics({ http, signal, query }),
+      bulkGet: (params, signal?: AbortSignal) => bulkGetCases({ http, signal, params }),
```

Comment:
Shouldn't this be `getCases` ?

Reply:
The `getCases` does a `find` operation. The bulk get is a `GET` operation for specific cases.

Reply:
Ahh okay, yeah it is confusing 😄